images/openwisp_openvpn/Dockerfile

```diff
@@ -1,6 +1,7 @@
+# hadolint ignore=DL3007
 FROM kylemanna/openvpn:latest
 
-RUN apk add --no-cache tzdata postgresql-client supervisor && \
+RUN apk add --no-cache tzdata~=2022a-r0 postgresql-client~=12.10-r0  supervisor~=4.2.0-r0 && \
```

Comment:
@nemesisdesign I forgot, what we concluded on locking these versions, in my experience, they don't change often, but when they do, we want to update them as quick as possible, mostly for security reasons.

Reply:
Oh, wait, I remember now!
Using something like `requirements.io` and ensuring it can capture all our deps (preferably automatically) will ensure I can update them timely and specially if there is security update. 

@codesankalp , what do you think about that? Do you think that will solve the problem? Should we do it in this PR or a fast follow?

Reply:
I would do in a follow up PR, this one is becoming very time consuming to review

Reply:
Yes we can use [snyk](https://snyk.io/) or something like that to update dependencies frequently.
I think doing this in another PR will be good.

Reply:
Yes, makes sense!
Let's create an issue for this.